src/modules/RootPart.rb

```diff
@@ -483,11 +482,16 @@ def RunFSCKonJFS(mount_type, device, error_message)
     # @return [String] nil on success, error description on fail
     def MountPartition(mount_point, device, mount_type, fsopts = "")
       if mount_type == ""
-        # storage-ng
-        # FIXME: knowing the fstab string for the filesystem type should be
-        # responsibility of the upcoming Y2Storage::FilesystemType (to be
-        # added as part of the "yast-storage-ng as a libstorage wrapper" change)
-        mount_type = probed.partitions.with(name: device).filesystems.first.to_s
+
```

Comment:
Please, don't delete the `storage-ng` marker from the comment. https://github.com/yast/yast-storage-ng/blob/master/doc/installer-hacks.md#temporarily-commented-code